deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -373,6 +373,10 @@ cephClusterSpec:
       osd:
         disabled: false
 
+  # Create an external rook-ceph cluster
+  external:
```

Comment:
This setting will likely be implemented similar to the monitoring setting for the cephcluster. You'll need to add a condition in the [cephcluster template](https://github.com/rook/rook/blob/master/deploy/charts/rook-ceph-cluster/templates/cephcluster.yaml#L7).

Reply:
Added the settings in template too,
But can I know how this template runs or take give the values to helm chart,

External I tried creating it using the k8s manifest way and it says,
`Error: INSTALLATION FAILED: failed to parse templates/cephcluster.yaml: error converting YAML to JSON: yaml: line 8: could not find expected ':'
`

Reply:
Helm evaluates the template yamls and replaces the variables. The template yamls will fail if you try to create them directly since the variables are invalid yaml and require the helm processing.